test/Ported/Issues/RavenDB_14109.ts

```diff
@@ -0,0 +1,49 @@
+import { IDocumentStore } from "../../../src/Documents/IDocumentStore";
+import { disposeTestDocumentStore, testContext } from "../../Utils/TestUtil";
+import { Company } from "../../Assets/Orders";
+import * as StreamUtil from "../../../src/Utility/StreamUtil";
+import { StreamQueryStatistics } from "../../../src/Documents/Session/StreamQueryStatistics";
+import { assertThat } from "../../Utils/AssertExtensions";
+
+describe("RavenDB_14109Test", function () {
+
+    let store: IDocumentStore;
+
+    beforeEach(async function () {
+        store = await testContext.getDocumentStore();
+    });
+
+    afterEach(async () =>
+        await disposeTestDocumentStore(store));
+
+    it.skip("queryStatsShouldBeFilledBeforeCallingMoveNext", async () => {
```

Comment:
skip?

Reply:
didn't manage to support that - we can add support in future - but this is rather minor